docs/configuration/cosmos-conf.rst

```diff
@@ -0,0 +1,56 @@
+Cosmos Config
+=============
+
+This page contains the list of all available Airflow configurations and environment variables specific to ``astronomer-cosmos`` that can be set in the airflow.cfg file or using environment variables.
+
+.. note::
+    For more information, see `Setting Configuration Options <https://airflow.apache.org/docs/apache-airflow/stable/howto/set-config.html>`_.
+
+**Sections:**
+
+- [cosmos]
+- [openlineage]
+
+[cosmos]
```

Comment:
Could you give an example of how the environment variable looks for one of these?

Reply:
I have added Default and Environment Variable for each config PTAL
<img width="760" alt="Screenshot 2024-05-20 at 4 19 01 PM" src="https://github.com/astronomer/astronomer-cosmos/assets/98807258/5450a229-9a40-44e6-a8a2-d1a75aeb33ee">